src/styles/blocks/categories.scss

```diff
@@ -0,0 +1,104 @@
+.categories {
+  &__wrapper {
+    display: grid;
+    row-gap: 40px;
+
+    @include onTablet {
+      row-gap: 56px;
+    }
+  }
+
+  &__link {
+    box-sizing: border-box;
```

Comment:
it s better to apply this property to all elements (use * selector)
when all elements have the same `box-sizing` value, their behavior becomes more predictable